### ISSUE TITLE
Themes: Midnight theme accent colors are broken

### ISSUE THREAD

**Issue by sarayourfriend** [Automattic/wp-calypso#49205]
### Steps to reproduce the behavior

Compare the [current Midnight theme](https://github.com/Automattic/wp-calypso/blob/HEAD/packages/calypso-color-schemes/src/shared/color-schemes/_midnight.scss) to the [pre-nav-unification midnight theme](https://github.com/Automattic/wp-calypso/blob/61acc957638fc7009632af4c0c058aa6bc0e82b7/packages/calypso-color-schemes/src/shared/color-schemes/_midnight.scss)

Note that the primary colors and accent colors have changed. Maybe this is intentional, but the accent color changed to an orange color without changing the `--color-accent-xx` values: 

https://github.com/Automattic/wp-calypso/blob/HEAD/packages/calypso-color-schemes/src/shared/color-schemes/_midnight.scss#L113

This causes some issues when composing the accent colors, for example with the search component’s box-shadow:

<img width="346" alt="Captura de Tela 2021-01-22 às 08 42 14" src="https://user-images.githubusercontent.com/24264157/105526792-b7e53100-5c97-11eb-943c-1f46783bfe82.png">

The inner ring, the orange one, is `--color-accent` while the outer ring is `--color-accent-10`.

I'm just guessing that this came from nav unification refactors of the themes, but I'm not sure. Feel free to untag nav-unification if it has nothing to do with it.

#### What I expected to happen
`--color-accent-xx` values should all be based off of `--color-accent`.

#### Browser / OS version
N/A

#### Is this specific to the applied theme? Which one?
Midnight

#### Does this happen on simple or atomic sites or both?
N/A

#### Is there any console output or error text?
N/A

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
Low

**Comment by getdave** [comment]
@frontdevde Would you be able to advise on this one in terms of

- Confirming the problem.
- Complexity and effort to fix.

Much appreciated.

**Comment by frontdevde** [comment]
We had to change the colors of the Calypso Midnight color scheme as there was one with the exact same name in wp-admin.

> Change Calypso Midnight color scheme to match Midnight in wp-admin - There are two very similar themes that share the same color scheme name: Midnight. As they are too close to be distinct color schemes, we will reduce them to the same color scheme. Staying consistent with core, we will change the Calypso Midnight color schemes to match the Midnight scheme in wp-admin. (Source: https://github.com/Automattic/wp-calypso/issues/45675#issue-702093124)

The wp-admin color scheme was used to generate a new Calypso color scheme. It does look strange that the other accent variable values aren't variations on the accent color does seem strange. I'll look into it in more detail to see why the accent values weren't changed and get back to you with details about effort required.

**Comment by retrofox** [comment]
I wonder why there are some `--blue` colors. Is that ok?

![image](https://user-images.githubusercontent.com/77539/112996020-30f28000-9142-11eb-9957-e14daa6ecd6e.png)

For instance, the box-shadow in wp-admin is #e14d43, while in the calypso app it's rgb(107, 191, 242) 
![image](https://user-images.githubusercontent.com/77539/112996797-e02f5700-9142-11eb-8400-e8041fe0c54f.png) vs ![image](https://user-images.githubusercontent.com/77539/112996650-bbd37a80-9142-11eb-8e0a-8b4b3039769b.png)

**Comment by retrofox** [comment]
@frontdevde have you had the chance to take a look at this one? I'm not sure how we should proceed to tackle the issue. Should we change the color palette manually?

**Comment by retrofox** [comment]
Hi, @cpapazoglou. How do you end up with this [color definition](https://github.com/Automattic/wp-calypso/blob/daa2bb3e7f04639c3acda6c1a5dced3a51bfcce8/packages/calypso-color-schemes/src/shared/color-schemes/_midnight.scss#L15-L30)?

```ec6
    {
      name: 'Midnight',
      default: 50,
      specs: {
        hue_start: 200,
        hue_end: 210,
        hue_curve: 'easeOutSine',
        sat_steps: [
          8, 27, 56, 80, 100, 100, 100, 100, 100, 100, 100, 100
        ],
        lum_steps: [
          96, 96, 95, 94, 87, 75.1, 61.6, 49, 40.5, 31, 20, 11
        ]
      }
    }
```

This definition generates a blue palette, which causes this color inconsistency.

```json
{
    "Midnight": "#b26200",
    "Midnight 0": "#f5ece6",
    "Midnight 5": "#f7dcc6",
    "Midnight 10": "#ffbf86",
    "Midnight 20": "#faa754",
    "Midnight 30": "#e68b28",
    "Midnight 40": "#d67709",
    "Midnight 50": "#b26200",
    "Midnight 60": "#8a4d00",
    "Midnight 70": "#704000",
    "Midnight 80": "#543100",
    "Midnight 90": "#361f00",
    "Midnight 100": "#1f1200"
}
```

Also, I'm afraid copying and pasting the core color won't work straightforward, because the notification-color #69a8bb generates blue shades, which are combined with the highlight-color #e14d43;

Notification color | Highlight color
--------|----------
![image](https://user-images.githubusercontent.com/77539/115418444-de036a00-a1cf-11eb-9477-6c073df04175.png) | ![image](https://user-images.githubusercontent.com/77539/115419831-0475d500-a1d1-11eb-82e1-bdfe9925bcef.png)

Do you know how we can deal with this?

**Comment by cpapazoglou** [comment]
> Hi, @cpapazoglou. How do you end up with this [color definition](https://github.com/Automattic/wp-calypso/blob/daa2bb3e7f04639c3acda6c1a5dced3a51bfcce8/packages/calypso-color-schemes/src/shared/color-schemes/_midnight.scss#L15-L30)?
> 
> ```
>     {
>       name: 'Midnight',
>       default: 50,
>       specs: {
>         hue_start: 200,
>         hue_end: 210,
>         hue_curve: 'easeOutSine',
>         sat_steps: [
>           8, 27, 56, 80, 100, 100, 100, 100, 100, 100, 100, 100
>         ],
>         lum_steps: [
>           96, 96, 95, 94, 87, 75.1, 61.6, 49, 40.5, 31, 20, 11
>         ]
>       }
>     }
> ```
> 
> This definition generates a blue palette, which causes this color inconsistency.
> 
> ```json
> {
>     "Midnight": "#b26200",
>     "Midnight 0": "#f5ece6",
>     "Midnight 5": "#f7dcc6",
>     "Midnight 10": "#ffbf86",
>     "Midnight 20": "#faa754",
>     "Midnight 30": "#e68b28",
>     "Midnight 40": "#d67709",
>     "Midnight 50": "#b26200",
>     "Midnight 60": "#8a4d00",
>     "Midnight 70": "#704000",
>     "Midnight 80": "#543100",
>     "Midnight 90": "#361f00",
>     "Midnight 100": "#1f1200"
> }
> ```
> 
> Also, I'm afraid copying and pasting the core color won't work straightforward, because the notification-color #69a8bb generates blue shades, which are combined with the highlight-color #e14d43;
> 
> Notification color	Highlight color
> ![image](https://user-images.githubusercontent.com/77539/115418444-de036a00-a1cf-11eb-9477-6c073df04175.png)	![image](https://user-images.githubusercontent.com/77539/115419831-0475d500-a1d1-11eb-82e1-bdfe9925bcef.png)
> Do you know how we can deal with this?

I would lie if I say I remember anything of this... Searched through this PR https://github.com/Automattic/wp-calypso/issues/47103 but still can't find how the definition was generated...

@mreishus is there a chance you remember?

**Comment by mreishus** [comment]
@cpapazoglou When I first ported ectoplasm over, there was the problem of wp-admin containing only 1 green color, but we needed several different shades of green in calypso. So I made the color studio thing here: https://github.com/Automattic/wp-calypso/pull/47122/commits/51136c1c909e26bd640d1a5a26804375b76052cc
Where those settings came from - I think I manually tweaked a existing color studio color set until it 'looked good'.

It looks like the [color studio stuff in midnight](https://github.com/Automattic/wp-calypso/commit/9fcfebc6bcf5574) is a tweaked version of the ectoplasm ones. Maybe it had the same problem (one color existing in wp-admin but we needed to show a variety of shades of that color in calypso)

Here is the Ectoplasm PR, which was the first color scheme ported which may have more clues about what we were thinking at the time: https://github.com/Automattic/wp-calypso/pull/47122
